dev/src/index.ts

```diff
@@ -640,6 +643,44 @@ export class Firestore {
     return new WriteBatch(this);
   }
 
+  /**
+   * Creates a [BulkWriter]{@link BulkWriter}, used for performing
+   * multiple writes in parallel. Gradually ramps up writes as specified
+   * by the 500/50/5 rule.
+   *
+   * @see [500/50/5 Documentation]{@link https://cloud.google.com/datastore/docs/best-practices#ramping_up_traffic}
+   *
+   * @private
+   * @param {object=} options BulkWriter options.
+   * @param {boolean=} options.disableThrottling Whether to disable throttling
+   * as specified by the 500/50/5 rule.
+   * @returns {WriteBatch} A BulkWriter that operates on this Firestore
+   * client.
+   *
+   * @example
+   * let bulkWriter = firestore.bulkWriter();
+   *
+   * bulkWriter.create(firestore.doc('col/doc1'), {foo: 'bar'})
+   *   .then(res => {
+   *     console.log(`Added document at ${res.writeTime}`);
+   *   });
+   * bulkWriter.update(firestore.doc('col/doc2'), {foo: 'bar'})
+   *   .then(res => {
+   *     console.log(`Updated document at ${res.writeTime}`);
+   *   });
+   * bulkWriter.delete(firestore.doc('col/doc3'))
+   *   .then(res => {
+   *     console.log(`Deleted document at ${res.writeTime}`);
+   *   });
+   * await bulkWriter.flush().then(() => {
```

Comment:
Nit: Make this `bulkWriter.close()` and remove line 678.

Reply:
done.